compiler/language_server/src/features_candy/analyzer/module_analyzer.rs

```diff
@@ -108,7 +108,8 @@ impl ModuleAnalyzer {
                     .unwrap();
                 let mut mir = (*mir).clone();
                 let mut static_panics = mir.static_panics();
-                static_panics.retain(|panic| panic.responsible.module == self.module);
+                static_panics
+                    .retain(|panic| -> bool { panic.responsible.module.clone() == self.module });
```

Comment:
The `module.clone()` doesn't seem to be necessary

```suggestion
                static_panics.retain(|panic| -> bool { panic.responsible.module == self.module });
```